src/capi/azext_capi/helpers/constants.py

```diff
@@ -7,3 +7,5 @@
 KUBECONFIG = "KUBECONFIG"
 AKS_INFRA_RG_NAME = "AKS_INFRA_RG_NAME"
 AKS_VNET_NAME = "AKS_VNET_NAME"
+CAPZ_BASE_CONTENT_URL = "https://raw.githubusercontent.com/kubernetes-sigs/cluster-api-provider-azure/release-1.9"
```

Comment:
this is bumping capz to release-1.9 in the process (we were using 1.7 before) which might break other things (specifically around the out of tree switch), would be safer to make that change separately

Reply:
moved back to 1.7